### DISCUSSION TITLE
new way to sign out for KillCurly without js on bookmarklets  even if chrome://settings/signOut is blocked by policy

### DISCUSSION THREAD

**Discussion by SIMONHSCH** [3kh0/ext-remover#1112] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
1 . drag [this](https://accounts.google.com/Logout?) in your bookmark bar
2 .go on chrome://newtab or just open a newtab
3 click the bookmark from step 1
4 hit ![Screenshot 2024-06-12 16 55 17](https://github.com/3kh0/ext-remover/assets/164594135/4d2d4d63-d834-4948-a95e-4baf4b20fa87)
5 click on cookies then on the button at the bottom of the pop-up
6 click the trash icon where it says "Google.com"

**Comment by S-PScripts** [comment]
when i do https://accounts.google.com/Logout? it just opens the accounts section in os settings

**Reply by SIMONHSCH** [reply]
is your URL bar like that ? :
![Screenshot 2024-06-13 16 38 11](https://github.com/3kh0/ext-remover/assets/164594135/41556e4d-ebd7-49b3-9a60-a9e49cb7cb20)

**Reply by S-PScripts** [reply]
yes.

**Reply by SIMONHSCH** [reply]
so you can do it :D

**Reply by S-PScripts** [reply]
oh you're ttalking about THAT i

**Reply by wvminecraftkid** [reply]
does it work on v123? trying and not working

**Reply by angelo641** [reply]
bro i have been tring to figure this out for 3 years now

**Comment by synapt0** [comment]
Could you please provide images or a video explaining this? I'm not sure what you mean.

**Reply by S-PScripts** [reply]
better guide

Killcurly without JS or SignOut:
1. Open a new tab.
3. Go to https://accounts.google.com/Logout? (Include the '?' in the link too!)
4. Hit the "i" to the left of the URL. (it's in the URL bar, next to the URL) 
5. Click "Cookies and site data".
6. Click "Manage on-device site data".
7. Click the Trash icon where it says "google.com".

**Reply by SIMONHSCH** [reply]
sorry if my version wsnt clear

**Reply by Brandon424-ops** [reply]
once you do this workaround don't you have to do the rest of killcurlys instructions like go to chrome://restart and go to add your school account back and so on???

**Reply by SIMONHSCH** [reply]
You can either go to Chrome://extension and click the "allow access to files URLs" on the Securly extension or do chrome://resteart  then after that you can go to [https://tinyurl.com/addSession](https://tinyurl.com/addSession) and signn in with your school account

**Reply by wvminecraftkid** [reply]
what if the tinyurl is blocked

**Reply by HelloIloveGimkit2** [reply]
here's a version that's way easier in case you want to watch youtube and incase it's managed

go to each site that is blocked due to your account
-> click the managed icon ( the one next to the url )
--> click cookies and site data
---> click that little manage smth smth smth then clear all the cookies relating to that site

**Comment by angelo641** [comment]
hi

**Comment by schoolexploitkid** [comment]
make a PR

**Comment by 88411** [comment]
Doesn't work cause go guardian blocks the page. When I open it using the link preview exploit no cookies appear on the site. Any ideas?

**Comment by schoolexploitkid** [comment]
doesn't work, it does not cause extensions to break.

**Reply by schoolexploitkid** [reply]
nvm, didnt restart ext

**Comment by Anty847** [comment]
For people that are wondering, you gotta follow the rest of KillCurly's instructions, that means you need to have chrome://restart unblocked, and the tinyurl unblocked.
